apps/condo/domains/common/constants/map.js

```diff
@@ -0,0 +1,14 @@
+
+// Key from examples of using 2gis mapgl
+const GUEST_API_KEY = 'bfd8bbca-8abf-11ea-b033-5fa57aae2de7'
```

Comment:
It's better to put this const inside the place of use. There is no way to reuse it anywhere else

Reply:
Done